File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer,Integer> map=new HashMap<>();
        int n=nums.length;
        for(int i=0;i<n;i++)
        {
            int ele=nums[i];
            int req=target-nums[i];
            if(map.containsKey(req))
            {
                return new int[]{map.get(req),i};
            }
            map.put(nums[i],i);
        }
        return new int[]{0,0};
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        mapp={}
        for i in range(len(nums)):
            ele=nums[i]
            req=target-nums[i]
            if req in mapp:
                return i,mapp[req]
            mapp[ele]=i
        return -1,-1

